cmd/main.go

```diff
@@ -32,7 +33,9 @@ func main() {
 	monitor := prometheus.NewMonitor("identity-admin-ui", logger)
 	tracer := tracing.NewTracer(tracing.NewConfig(specs.TracingEnabled, specs.JaegerEndpoint, logger))
 
-	router := web.NewRouter(tracer, monitor, logger)
+	hClient := ih.NewClient(specs.HydraAdminURL)
```

Comment:
small tiny bit here (which u haven't been exposed before)

we gotta do a check on this env config var, if empty all falls apart, where we do it can differ:

* use annotations (should be doable) and make it not nullable
* simply `if specs.HydraAdminURL ==  "" {panic("empty url")}`
* panic inside the `ih.NewClient`

basically panic and fail hard and fast